client/elife-theme/src/elements/ui/Steps.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export default css`
+  margin-bottom: 100px;
```

Comment:
is this the value from the invision prototypes?

Reply:
nope - its a guess

Reply:
an easily changeable guess (more of these plz :) )

Reply:
but i think the correct value is annotated?

Reply:
yes, it's 48px